app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -226,4 +227,20 @@ class PatientsScreenController @Inject constructor(
         .filter { it.permissionResult == RuntimePermissionResult.GRANTED }
         .map { Ui::openScanSimpleIdCardScreen }
   }
+
+  private fun handleVisibilityOfSyncIndicator(events: Observable<UiEvent>): Observable<UiChange> {
```

Comment:
What do you think of `handleVisibilityOfSyncIndicator` -> `showSyncIndicatorIfUserCanSyncData` ?

The term `handle` is often abused because it's very generic